rclcpp_lifecycle/CMakeLists.txt

```diff
@@ -159,3 +159,8 @@ ament_export_targets(${PROJECT_NAME})
 ament_export_dependencies(lifecycle_msgs rcl rclcpp rcl_interfaces rcl_lifecycle rcutils rosidl_typesupport_cpp)
 
 ament_package()
+
+if(TEST cppcheck)
+  # must set the property after ament_package()
+  set_tests_properties(cppcheck PROPERTIES TIMEOUT 360)
+endif()
```

Comment:
It shouldn't be necessary to have this after the `ament_package`, and in fact I would prefer if this was in the `if(BUILD_TESTING)` block.  See https://github.com/ros2/rclpy/blob/995bbed81cdb6bbcc2910c23ebfb280f8564c85f/rclpy/CMakeLists.txt#L144-L146 for an alternate way to do this.

Reply:
I took that from `rclcpp`:

https://github.com/ros2/rclcpp/blob/2bb1dc2c716ae12a24145891724f8436aa4e52bd/rclcpp/CMakeLists.txt#L282

Reply:
Im adding a commit to make it equal to the rclpy one.